Ventivader/Ventivader.xcodeproj/project.pbxproj

```diff
@@ -602,14 +598,14 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 93TH93CMJB;
+				DEVELOPMENT_TEAM = 54L86QB3ZV;
```

Comment:
not sure we need to change the development team. can you please take a look?

Reply:
This is the Signing Settings, I need to change the team to run it in a real device. 
I will do a new PR (and close this), as Lu sent me a new icon, and i will make sure this DEVELOPMENT_TEAM wont get modified